addons/web_editor/security/ir.model.access.csv

```diff
@@ -1,3 +1,7 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_web_editor_converter_test,access_web_editor_converter_test,model_web_editor_converter_test,,1,1,1,1
 access_web_editor_converter_test_sub,access_web_editor_converter_test_sub,model_web_editor_converter_test_sub,,1,1,1,1
+access_web_editor_field_history_all,access.web.editor.field.history.all,model_field_html_history_diff,,0,0,0,0
+access_web_editor_field_history_portal,access.web.editor.field.history.portal,model_field_html_history_diff,base.group_portal,1,0,0,0
```

Comment:
Seems this will quickly be a security issue.

Reply:
I will need to rework the acces rules indeed. I just add those to be able to make it work, and din't thought them truth yet. :ok_hand: